README.md

```diff
@@ -34,37 +34,77 @@ Concern has been raised that AWS restricts what they allow to be pentested (thei
 - References:
   - https://cloud.google.com/resource-manager/docs/organization-policy/restricting-service-accounts#disable_service_account_default_grants
 
+### AWS: Signature version 1 (SigV1) is insecure
+- Summary: When making authenticated API requests to AWS, the requests must be signed with your
+AWS access key. The initial signing algorithm, SigV1, was vulnerable to collisions. A person-in-the-middle attack would be able to modify signed requests via specially constructed collisions.
+- Platform: AWS
+- Severity: Medium
+- Date: until December 18th, 2008
+- Discoverer: Colin Percival
+- Customer action: N/A, SigV1 is deprecated at this point
+- References:
+  - http://www.daemonology.net/blog/2008-12-18-AWS-signature-version-1-is-insecure.html
+
+### AWS: AWS published official AMIs with recoverable deleted files
+- Summary: Researchers, while investigating the security posture of Public AMIs, were able to undeleted 8,996 files from an official image that was published by Amazon AWS.
+- Platform: AWS
+- Severity: Low
+- Date: June, 2011
+- Discoverer: Marco Balduzzi, Jonas Zaddach, Davide Balzarotti, Engin Kirda, Sergio Loureiro
+- Customer action: Follow [best practices](https://aws.amazon.com/articles/how-to-share-and-use-public-amis-in-a-secure-manner/) when sharing Public AMIs
+- References:
+  - http://seclab.nu/static/publications/sac2012ec2.pdf
+
 ### AWS: Bypasses in IAM policies and over-privileged
 - Summary: Repeated examples of AWS provided managed policies or guidance in documentation for policies with mistakes that allow the policies to by bypassed. Generically, there are also over-privileged policies and policies with spelling mistakes and other issues.
 - Platform: AWS
 - Severity: Medium
 - Date: November 17, 2017 (date and discoverer is of the first issue, references provide other examples by other individuals)
 - Discoverer: Scott Piper, Duo Labs + Summit Route
 - Customer action: Review the policies provided by AWS
-- References: 
+- References:
   - https://duo.com/blog/potential-gaps-in-suggested-amazon-web-services-security-policies-for-mfa
   - https://summitroute.com/blog/2019/06/18/aws_iam_managed_policy_review/
   - https://medium.com/ymedialabs-innovation/an-aws-managed-policy-that-allowed-granting-root-admin-access-to-any-role-51b409ea7ff0
 
+### Azure: CSV Injection in Activity Log
+- Summary: An Azure user can attempt (but fail) to create an Azure “Resource Group” with a malicious name.  In setting this name as a Microsoft Excel formula, the system would reject the syntax (due to certain blacklisted characters) but still log the original string in the Azure “Activity Log”.  When a victim user downloaded the – now infected – CSV logs and opened them in Excel, the attacker’s remote code was executed, compromising the user (and likely their Azure account as well).  The single set of Activity Logs also means a lower-privileged user could use this attack to target administrative users.
```

Comment:
I'm hesitant on this one and the CloudTrail one that is similar as it's more of an issue with Excel, so I may remove this in the future, but thank you for point it out.